Classes/Domain/Factory/CommentFormFactory.php

```diff
@@ -0,0 +1,113 @@
+<?php
+declare(strict_types = 1);
+
+/*
+ * This file is part of the package t3g/blog.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace T3G\AgencyPack\Blog\Domain\Factory;
+
+use T3G\AgencyPack\Blog\Domain\Finisher\CommentFormFinisher;
+use T3G\AgencyPack\Blog\Domain\Validator\GoogleCaptchaValidator;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+use TYPO3\CMS\Extbase\Utility\LocalizationUtility;
+use TYPO3\CMS\Extbase\Validation\Validator\EmailAddressValidator;
+use TYPO3\CMS\Extbase\Validation\Validator\NotEmptyValidator;
+use TYPO3\CMS\Extbase\Validation\Validator\StringLengthValidator;
+use TYPO3\CMS\Extbase\Validation\Validator\UrlValidator;
+use TYPO3\CMS\Form\Domain\Configuration\ConfigurationService;
+use TYPO3\CMS\Form\Domain\Factory\AbstractFormFactory;
+use TYPO3\CMS\Form\Domain\Finishers\RedirectFinisher;
+use TYPO3\CMS\Form\Domain\Model\FormDefinition;
+use TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController;
+
+class CommentFormFactory extends AbstractFormFactory
+{
+    /**
+     * Build a FormDefinition.
+     * This example build a FormDefinition manually,
+     * so $configuration and $prototypeName are unused.
+     *
+     * @param array $configuration
+     * @param string $prototypeName
+     * @return FormDefinition
+     */
+    public function build(array $configuration, string $prototypeName = null): FormDefinition
+    {
+        $prototypeName = 'standard';
```

Comment:
This was taken out of the example, not sure if we should really ignore the incoming `$prototypeName`.